src/main/webapp/js/components/interface/drawer/DrawerButton.js

```diff
@@ -16,7 +16,8 @@ define(function (require) {
             super(props);
             this.state = {
                 mouseOver: false,
-                buttonActived: false}
+                buttonActived: false
```

Comment:
@ddelpiano what's `actived`? I know it was pre-existing but we might as well rename it :)